src/components/Footer.js

```diff
@@ -32,7 +32,7 @@ const navigation = [
 
 export default function Footer() {
   return (
-    <footer className="bg-gray-800 mt-auto">
```

Comment:
I think there are now 2 `mt-`, so the `mt-5` will override the `mt-auto`, can `mt-auto` be removed? Please test it again after the change

Reply:
Thanks for pointing that out, I've removed the `mt-auto` class and everything looks good now.

Reply:
ok great, thank you @Bridgetamana 👍